pkg/k8s/network_policy.go

```diff
@@ -49,15 +48,16 @@ func GetPolicyLabelsv1(np *networkingv1.NetworkPolicy) labels.LabelArray {
 		return nil
 	}
 
-	policyName := np.Annotations[annotation.Name]
+	policyName := np.Name
```

Comment:
Revert this change and make it in a new PR .

Reply:
Done.